files/BonnPC.md

```diff
@@ -6,7 +6,7 @@ Here are some tips for setting up the computer.
 ## Language
 
 * You need to login with your `@math.uni-bonn.de` username and password.
-* The keyboard layout for the login screen is German. This cannot be changed. 
+* The keyboard layout for the login screen is German. This cannot be changed. (This applies both to the first login after booting the computer, and to later log-ins after locking the screen.)
```

Comment:
This is false: if I lock my screen there is a button to change the keyboard language to english.

Reply:
Oh! That's a pleasant surprise. (I remember stumbling across this detail in Berlin.) Did you test if it works?

Reply:
If so, I'll happily add a not to that sentence.

Reply:
Yes, I use it frequently (and it either remembers that I chose English, or chooses English (my OS language) as default).

Reply:
```suggestion
* The keyboard layout for the login screen is German. This cannot be changed. (This does *not* apply after locking the screen: there, you have the option to change the keyboard language.)

Reply:
```suggestion
* The keyboard layout for the login screen is German. This cannot be changed. (This does *not* apply to entering your password after locking the screen: there, there you have the option to change the keyboard language.)
```